templates/candlepin.service.erb

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Candlepin
+Wants=syslog.service
+
+[Service]
+Restart=always
+RestartSec=30
+TimeoutStartSec=0
+TimeoutSec=300
+ExecStartPre=-/usr/bin/podman pull quay.io/foreman/candlepin:<%= scope['candlepin::ssl_port'] %>
```

Comment:
I think you intended `candlepin_version` here.

Reply:
This makes it harder to run Katello inside a disconnected or offline environment. I'm fine with it considering:

- this is optional
- it's new and somewhat experimental (at least to my eyes).

Just something to be aware of

Reply:
Truish on the more difficult. You can pull from a locally stored directory with podman which I how I'd envision disconnected working with containers. I tend more so agree that this is a nicety of systemd that it can pull on service start for us but we should prefer to pull the images in some other place to have more control.